scripts/GetClrDbg.ps1

```diff
@@ -104,13 +104,20 @@ if (-not $RuntimeID) {
 }
 Write-Host "Info: Using Runtime ID '$RuntimeID'"
 
-# create the install folder if it does not exist
-if (-not (Test-Path -Path $InstallPath -PathType Container)) {
-    New-Item -ItemType Directory -Force -Path $InstallPath
+# create the temp folder if it does not exist
+$GuidString = [System.Guid]::NewGuid()
+$TempPath = Join-Path -Path $env:TEMP -ChildPath $GuidString
```

Comment:
done. 

---
In reply to: [63798580](https://github.com/Microsoft/MIEngine/pull/332#discussion_r63798580) [](ancestors = 63798580)